Library/Homebrew/dev-cmd/create.rb

```diff
@@ -23,6 +23,8 @@ def create_args
              description: "Create a basic template for an Autotools-style build."
       switch "--cmake",
              description: "Create a basic template for a CMake-style build."
+      switch "--go-mod",
```

Comment:
What about just making this `--go`?

Reply:
Not every Go project uses modules. Some projects needs to be in GOPATH to build successfully. In fact I would say that the `--go` option would be more appropriate in the latter case.

Reply:
Nothing that's generated or put in the formula is doing anything with modules, though?

Reply:
Fair points. Changes committed.